env/staging/performance.yaml

```diff
@@ -1,36 +1,61 @@
+####################################### RESOURCE REQUESTS #######################################
+# PRIMARY CELERIES
+
 apiVersion: apps/v1
 kind: Deployment
 metadata:
   labels:
-    app: api
-  name:  api
```

Comment:
ah, there's some reorg going on where api is now down below. Copied the prod?

Reply:
Yeah I moved everything around to keep it somewhat organized and consistent..

Reply:
consistent is good